Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -216,13 +216,46 @@
   - type: Projectile
     damage:
       types:
-        Heat: 5
+        Shock: 1
     soundHit:
       path: "/Audio/Weapons/Guns/Hits/taser_hit.ogg"
     forceSound: true
+  - type: TimedDespawn
+    lifetime: 0.170 # Very short range
+  - type: StunOnCollide
+    stunAmount: 0
+    knockdownAmount: 2.5 # Enough to subdue and follow up with a stun batong
+    slowdownAmount: 2.5
+    walkSpeedModifier: 0.5
+    sprintSpeedModifier: 0.5
+
+- type: entity
+  name: taser bolt
+  id: BulletTaserSuper
+  description: If you can see this, you've probably been stun-meta'd
+  parent: BulletTaser
+  categories: [ HideSpawnMenu ]
+  components:
```

https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions
```suggestion
- type: entity
  parent: BulletTaser
  id: BulletTaserSuper
  categories: [ HideSpawnMenu ]
  name: taser bolt
  description: If you can see this, you've probably been stun-meta'd
  components:
```